content/api/servo-assembly/arduino-commands.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Custom Arduino Commands
+---
+
+At the core of the servo assembly is an [Arduino Uno][arduino]. By default the
+Arduino runs firmware created by SourceBots providing simple access to the
+attached servo shield, compatible ultrasound sensors and to the GPIO pins
+directly.
+
+You can also extend the firmware on the Arduino to support more features, for
+example if you have other types of sensor which require higher precision timing
+than can be achieved through the Python API.
+
+## Requirements
+
+As your new firmware will still need to be able to talk to the Python code
+running on the Raspberry Pi, you'll need to base it on SourceBots default
+firmware. You can get a copy of the source code from GitHub either by
+downloading the [repo archive][servo-firmware-zip] or by cloning the repo:
```

Comment:
Perhaps here, rather than specifically getting a zip or using `git clone`, we just link to the repo. This means they can not only see the source without downloading it, but the discovery path for clone / download instructions is very straightforward

Reply:
e1dc43f